postgresql/examples/secrets/postgres-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
any reason why that is preferred? I have seen both and didnt know which one to choose

Reply:
I like it for these reasons:
- the data in the secret is managed in plain text (but encripted by strongbox)
- the secret can be overwritten easily through kustomization
- when using nameSuffixHash, the deployment/stateful set will restart automatically if the secret changes.

Reply:
Also it would be good to include this secret generator in the /manifests/kustomization.yaml with a template, so that the clients will know what to put there.

Reply:
@sbuliarca  I think I have done what we need to for that!